Formula/fontforge.rb

```diff
@@ -16,71 +16,72 @@ class Fontforge < Formula
 
   deprecated_option "with-gif" => "with-giflib"
 
-  # Autotools are required to build from source in all releases.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  if build.head?
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+
+    resource "gnulib" do
+      url "git://git.savannah.gnu.org/gnulib.git",
+        :revision => "29ea6d6fe2a699a32edbe29f44fe72e0c253fcee"
+    end
+  end
+
+  depends_on :x11 => :optional
```

Comment:
Uh, FontForge really does depend on X11. Without support for things like this, it's near impossible to get anyone on Mac to actually test new features with any reasonable sort of speed. X11 support has already been disabled by default, so there's not much in the way of having to support anything.